tests/epyccel/test_epyccel_generators.py

```diff
@@ -244,4 +244,12 @@ def f():
         return v
     
     f_epyc = epyccel(f, language = language)
+
+def test_sum_with_two_variables(language):
+    def f():
+        x = sum(i-j for i in range(10) for j in range(7))
+        return x
+
+    f_epyc = epyccel(f, language=language)
+
     assert f() == f_epyc()
```

Comment:
You seem to have lost your assert
```suggestion
    f_epyc = epyccel(f, language = language)

    assert f() == f_epyc()

def test_sum_with_two_variables(language):
    def f():
        x = sum(i-j for i in range(10) for j in range(7))
        return x

    f_epyc = epyccel(f, language=language)

    assert f() == f_epyc()
```